paper.md

```diff
@@ -269,9 +266,11 @@ The general paradigm for building reproducible workflows with Popper usually con
   b. If a prebuilt image is not available, a `Dockerfile` can be used to build an image manually which is a file containing specifications for building Docker images.
 3. Running the workflow and refining it.
 
+[^artifacts]: <https://github.com/ivotron/popper2-paper/tree/master/artifacts>
```

Comment:
I've renamed the repo `popper-canopie-paper`.

Also, we can link to the repository instead of the `artifacts` folder. Since we'll work on #42, that should be enough for people to find info on how to find the code of the workflows.